packages/core-mobile/app/navigation/wallet/EarnScreenStack/EarnScreenStack.tsx

```diff
@@ -35,64 +36,66 @@ const EarnStack = createStackNavigator<EarnStackParamList>()
 
```

Comment:
these intro popups are being stretched too long and there is no spacing between it and the bottom tab. you can see the figma design for it [here](https://www.figma.com/design/6ETlkl6Gi5cMvSRIvoRP6y/Mobile?t=yZldRWlY3NisUFHF-0) 
<img width="810" alt="Screenshot 2024-08-26 at 5 04 04 PM" src="https://github.com/user-attachments/assets/b0172220-14e4-429d-bbe7-a4fdde0c1279">